### DISCUSSION TITLE
Cannot add contributors to a private repo

### DISCUSSION THREAD

**Discussion by garajuria** [community/community#81613] [New to GitHub]
Hello,

I am trying to add some contributors to a private repo and it says "User could not be added" with any further info. Does anyone have any clue on how to solve it?

![image](https://github.com/community/community/assets/154506029/149ca7c6-541e-4cd4-bfa6-3c960609127a)

Thank you in advance!

**Comment by queenofcorgis** [comment]
Hi there @garajuria 👋🏼 , 

May I suggest checking out [this discussion](https://github.com/orgs/community/discussions/35582) first? I'm not sure if that could be the issue, but a good first place to look.
